public_website/templates/public_website/resources.html

```diff
@@ -740,6 +740,15 @@ <h2 class="resources-section-title">Books for adults</h2>
                     <p class="resource-sub-title">758 questions from children answered by the late Dr. A. P. J. Abdul Kalam, 11th President of India.</p>
                 </div>
             </div>
+            <div class="resource-weblink">
+                <a href="https://www.youtube.com/channel/UCST7O07V0HwBkK_wCB6EB5g/playlists" target="_blank">
+                    <img src="{% static 'hc-verma-weblinks-resource-page.JPG' %}" alt="hc verma screenshot">
```

Comment:
The link to the image should be 'resources/<image-name>.jpg'

Reply:
Thanks, @nitinpaul. I have made the required commit. Please check it.